lib/src/utils/constants.dart

```diff
@@ -0,0 +1,61 @@
+import 'package:flutter/material.dart';
+
+successAlert(BuildContext context) {
```

Comment:
Should we include return values on these methods?
```suggestion
Future<void> successAlert(BuildContext context) {
```

Reply:
adding the future return type without the return key word throws a body might complete normally error
Otherwise, it shows no errors if the return type isnt specified and the return key word is removed

Reply:
Would you be able to add `Future<void>` and still keep the `return`? I guess in this case it's not a big deal because this method is only used internally, but I prefer not to create functions with `dynamic` return type!
```dart
Future<void> showErrorDialog() {
  return showDialog(
    context: context,
    builder: (BuildContext context) {
      return const AlertDialog(
        title: Text('some error message'),
        contentTextStyle: TextStyle(
          backgroundColor: Colors.red,
        ),
      );
    },
  );
}
```